recipes/fermikit/test-data/test.fastq

```diff
@@ -0,0 +1,100 @@
+@phiX174_29_494_0_1_0_0_2:0:0_3:0:0_0
```

Comment:
I don't think a reasonable test can be written without a fastq file, you'd probably not detect runtime dependency problems. And bioconda would hugely benefit from this practice. My feeling is that a good fraction of packages here are broken (perl, bioperl, fastx, gnuplot among the ones I recently came across). Functional tests would be a good way to notice this early. I can do the simply help-text grep thing if you prefer, but I'd strongly advocate small functional tests if possible.